### DISCUSSION TITLE
Windows 11 + Server 2022 Compatibility

### DISCUSSION THREAD

**Discussion by AndrewPla** [Windos/BurntToast#208] [General]
I noticed that in the readme it says "PowerShell Module for displaying Windows 10 and Windows Server 2019 Toast Notifications"

Is there anything preventing this module from working with Server 2022 or Windows 11? In my limited testing, the module appears to work. 

If this works with Windows 11 and Server 2022 I am more than happy to submit a PR with doc changes, but wanted to run it by first.

**Comment by tzitman** [comment]
++ @Windos 

also, noticed:
```Powershell
New-BTContextMenuItem # for right-click menu 
```

on Windows Server 2019 doesn't seem to show anything when right-clicking the toast, while it works correctly on Windows Server 2022.

On Windows Server 2016, the same situation occurs as with Server 2019, plus it doesn't have a header at all using:
```Powershell
$header = New-BTHeader -Title $headLine
```

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
